.github/mergify.yml

```diff
@@ -4,7 +4,6 @@ pull_request_rules:
       - author~=^dependabot(|-preview)\[bot\]$
       - check-success~=Lint
       - check-success~=Test
-      - check-success~=Build and push
```

Comment:
is this strict enough to ensure that both the frontend and backend can build before the upgrade can be merged?

Reply:
Added one more step for linting, please review